APIs/IPMXAPI.raml

```diff
@@ -0,0 +1,91 @@
+#%RAML 1.0
+
+title: IPMX Device Control
+baseUri: http://example.api.com/x-nmos/ipmx/{version}
+version: v1.0
+mediaType: application/json
+
+/senders:
+  displayName: Senders
+  get:
+    description: Sender root
+    responses:
+      200:
+        description: List available Senders. All UUIDs are appended with / to indicate that the ID forms part of the URL for the next layer down.
+  /{senderId}:
+    get:
+      description: List all the API endpoints for a single sender.
+      responses:
+        200:
+          description: Returned when success.
+        404:
+          description: Returned when the requested Sender ID does not exist.
```

Comment:
I do not think we need to indicate the 404s they are spec'd in IS-04 if I recall

https://github.com/AMWA-TV/nmos-device-connection-management/blob/72c64a9609776b83c56f057119bfed7bbefe4140/APIs/ConnectionAPI.raml#L127-L131

Reply:
@N-Nagorny - you agree on this one? If so we can strike it.

Reply:
@prince-chrismc, maybe I don't quite understand what you mean.
A user always can request /senders/{senderId} for non-existing ID. What should we return if not 404?
That's true, it is specified in IS-04, and the hyperlink points to IS-05 where 404 for this case is also present, so it looks like a reason to have it here as well.
Could you clarify your point, please?

Reply:
> hyperlink points to IS-05 where 404 for this case is also present, so it looks like a reason to have it here as well.

You are absolutely correct, my mistaken. I did not see it was defined lower down 🙊 I was not looking at the correct end point.

https://github.com/AMWA-TV/nmos-device-connection-management/blob/72c64a9609776b83c56f057119bfed7bbefe4140/APIs/ConnectionAPI.raml#L144